midp/media.js

```diff
@@ -753,7 +788,7 @@ AudioRecorder.prototype.stop = function() {
             // The audio data we received are encoded with a proper format, it doesn't
             // make sense to concatenate them like the socket, so let just override
             // the buffered data here.
-            this.data = new Uint8Array(message.data);
```

Comment:
I neglected that there are other types except 3gp can be recorded.  I fixed this with my last commit ` e1be4e6`.  It will the check the required file format before doing 3gp to amr conversion.